app.py

```diff
@@ -7,12 +7,14 @@
 from bson.objectid import ObjectId
 from mongoengine import connect
 import dateutil.parser
+import bcrypt
 import traceback
 
 app = Flask(__name__)
 client: MongoClient = MongoClient("localhost:27017")
 
 db = client.user  # need for non graphql routes that access db
+cdb = client.users # need for non graphql routes that access db
```

Comment:
could we not create a new db, but use the old one and create a new collection in it, like 'users' ? so down in the routes you would do like db.users.insert_one, etc, just because we already have a collection in db called people so it's kind of confusing